### PR TITLE
Tag PAINTER v0.4.0

### PR DESCRIPTION
**Pull request by andferrari**

### CODE REVIEW

File: PAINTER/versions/0.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.4 0.5
```
Comment (andferrari) [inline PAINTER/versions/0.4.0/requires]: The only problem with  Julia 0.5 is that `PAINTER.jl` will not run in parallel due to tknopp/NFFT.jl#17

We have removed the version constraint on julia and added a warning in the doc.

Comment (tkelman) [inline PAINTER/versions/0.4.0/requires]: best to remove it here from the `PAINTER/versions/0.4.0/requires` metadata file as well

Comment (andferrari) [inline PAINTER/versions/0.4.0/requires]: great! I will wait

Comment (robertdj) [inline PAINTER/versions/0.4.0/requires]: My METADATA fork is waiting for another approval. Hopefully that will be accepted soon and then I'll make a new release for NFFT.

**Comment by andferrari** [comment]
Everything has been fixed now. 

`requires` has been changed. Do I need to create a new pull request ?

**Comment by tkelman** [comment]
or you could modify the files in place in this branch of your fork

**Comment by andferrari** [comment]
Thanks. I will  create a new pull request.

**Pull request closed without merging**